common/src/main/java/com/microsoft/identity/common/adal/internal/AuthenticationConstants.java

```diff
@@ -1417,6 +1433,7 @@ public enum API {
             SIGN_OUT_FROM_SHARED_DEVICE(MSAL_SIGN_OUT_FROM_SHARED_DEVICE_PATH, null, VERSION_3),
             GENERATE_SHR(GENERATE_SHR_PATH, null, VERSION_6),
             BROKER_HELLO(BROKER_API_HELLO_PATH, BROKER_VERSION_1, null),
+            BROKER_UPLOAD_LOGS(BROKER_API_UPLOAD_LOGS, BROKER_VERSION_4, null),
```

Comment:
New enum constants are meant to be added at the end of the list because enums are ordinals under the hood and the values for existing constants mustn't be changed.

Reply:
Oops yes! I missed this. Added it now.